### ISSUE TITLE
Technique C40: description and links mention text-shadow but examples use box-shadow

### ISSUE THREAD

**Issue by moloko** [w3c/wcag#3055]
In [Technique C40](https://www.w3.org/WAI/WCAG21/Techniques/css/C40q) the description text (and example links' text) both reference `text-shadow` but the examples actually use `box-shadow`:

![screenshot showing the issue](https://user-images.githubusercontent.com/1249597/220696753-557bae72-21c7-41fd-93aa-cb6fd8143018.png)

**Comment by moloko** [comment]
@fstrr the link text for the examples look like they could benefit from being updated to be more specific, currently they both have the following text:
> Working example of combining a dark outline and light text shadow

But it would be clearer if the first link was changed to:
> Working example of a light and dark dotted indicator

And the second to something like:
> Working example of a thicker light and dark indicator

**Comment by moloko** [comment]
Oh, and both examples have the word 'Description' which I guess should either be removed - or replaced with an actual description...
![image](https://user-images.githubusercontent.com/1249597/220703449-bb5d52b6-3f96-4e1f-a9f9-24f91b289203.png)

**Comment by fstrr** [comment]
@moloko Thanks for these. I was updating that document last night and missed those odd "description" paragraphs. I've removed them. I've also updated the link text.

FYI, the dotted border doesn't conform to the proposed new Focus Appearance criterion (see also #2469), so I've updated that example to used a 2px dashed border instead.

Not sure when these changes will be merged in, but they're there now.
